src/css/flags/fullscreen.less

```diff
@@ -10,7 +10,7 @@
         margin: 0;
         position: fixed;
 
-        &.jw-flag-user-inactive {
+        &.jw-flag-user-inactive .jw-media {
```

Comment:
Media may not fill the player container, so it may be safer to target both player and media containers:
```css
&.jw-flag-user-inactive, &.jw-flag-user-inactive .jw-media {
```

@jfcherng would this still work?